config/development.toml

```diff
@@ -277,6 +277,7 @@ debit = { currency = "USD" }
 [tokenization]
 stripe = { long_lived_token = false, payment_method = "wallet", payment_method_type = { type = "disable_only", list = "google_pay" } }
 checkout = { long_lived_token = false, payment_method = "wallet" }
+mollie = {long_lived_token = false, payment_method = "card"}
```

Comment:
Please make the corresponding configuration changes in `config/config.example.toml`, `config/docker_compose.toml` and `loadtest/config/development.toml`.